src/pages/ResetPassword/index.jsx

```diff
@@ -9,10 +9,14 @@ import * as yup from 'yup';
 import { useFormatMessage } from 'hooks';
 import { resetPassword, authCleanUp } from 'state/actions/auth';
 import paths from 'pages/Router/paths';
-import errorMessage from 'components/ErrorMessage';
+import ErrorMessage from 'components/ErrorMessage';
```

Comment:
In some components we use relative imports and in others we don't. Can we do all the imports as relative?